server/src/api/modules/groups/group.routes.ts

```diff
@@ -26,6 +26,7 @@ api.get('/:id/records', setupController(controller.records));
 api.get('/:id/hiscores', setupController(controller.hiscores));
 api.get('/:id/name-changes', setupController(controller.nameChanges));
 api.get('/:id/statistics', setupController(controller.statistics));
+api.get('/:id/activities', setupController(controller.activities));
```

Comment:
can you change this to `/:id/activity` instead? (and update docs, controller and routes)